test/benchmark_tests/001-atlas/main

```diff
@@ -0,0 +1,26 @@
+
+cvmfs_test_name="ATLAS benchmark"
+cvmfs_test_autofs_on_startup=false
+cvmfs_benchmark="yes"
+
+FQRN=atlas.cern.ch
+
+
+cvmfs_run_benchmark() {
+  export ATLAS_LOCAL_ROOT_BASE=/cvmfs/atlas.cern.ch/repo/ATLASLocalRootBase
```

Comment:
Can you check if the test runs without `ATLAS_LOCAL_ROOT_BASE` or perhaps with `local ATLAS_LOCAL_ROOT_BASE=...`.  That would avoid a possible source of environment pollution.